jsons/parameters.json

```diff
@@ -15897,5 +16149,840 @@
+  },
+  {
+    "_id": 393,
+    "default_value": 0,
+    "description": "This parameter controls how the peptide_mass_tolerance parameter is applied. That tolerance can be applied to the singly charged peptide mass (default=0) or it can be applied to the precursor m/z (1).",
+    "key_translations": {
+      "comet_style_1": "precursor_tolerance_type",
+      "ursgal_style_1": "precursor_tolerance_type"
+    },
+    "name": "precursor_tolerance_type",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 394,
+    "default_value": 0,
+    "description": "This parameter specifies how theoretical fragment ion peaks are represented.",
+    "key_translations": {
+      "comet_style_1": "theoretical_fragment_ions",
+      "ursgal_style_1": "theoretical_fragment_ions"
+    },
+    "name": "theoretical_fragment_ions",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 395,
+    "default_value": [
+      600.0,
+      8000.0
+    ],
+    "description": "Defines the mass range of peptides to search (based on MH+ or the singly protonated mass).",
+    "key_translations": {
+      "comet_style_1": "digest_mass_range",
+      "ursgal_style_1": "digest_mass_range"
+    },
+    "name": "digest_mass_range",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "list"
+  },
+  {
+    "_id": 396,
+    "default_value": 20,
+    "description": "This parameter defines the maximum number of proteins (identifiers/accessions) to report. If set to -1, there will be no limit on the number of reported additional proteins.",
+    "key_translations": {
+      "comet_style_1": "max_duplicate_proteins",
+      "ursgal_style_1": "max_duplicate_proteins"
+    },
+    "name": "max_duplicate_proteins",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 397,
+    "default_value": 0,
+    "description": "This parameter sets the maximum indexed database search run time for a scan/query.",
+    "key_translations": {
+      "comet_style_1": "max_index_runtime",
+      "ursgal_style_1": "max_index_runtime"
+    },
+    "name": "max_index_runtime",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 398,
+    "default_value": 100,
+    "description": "This parameter controls the number of peptide search results that are stored internally. Depending on what post-processing tools are used, one may want to set this to the same value as num_output_lines (id193).",
+    "key_translations": {
+      "comet_style_1": "num_results",
+      "ursgal_style_1": "num_results"
+    },
+    "name": "num_results",
+    "tag": [],
+    "triggers_rerun": false,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 399,
+    "default_value": 0,
```

Comment:
Integers should only be used if the value is indeed an integer. If 0 stands for something, then the something should be the value and 0 the translation.

Reply:
Totally true. Updated it with the correct value translations dict.